### ISSUE TITLE
rollup_bundle should load rollup-plugin-json

### ISSUE THREAD

**Issue by hperl** [bazelbuild/rules_nodejs#626]
# 🚀 feature request

### Relevant Rules

`rollup_bundle`

### Description

In `rollup_bundle`, if I import a package which loads a JSON file via `import` rollup complains with
```
rollup Error: Unexpected token (Note that you need rollup-plugin-json to import JSON files)
```

For example, this happens in `@grpc/grpc-js` via
```typescript
import grpc from "@grpc/grpc-js";
```

### Describe the solution you'd like

I would like for `rollup_bundle` to automatically make available the plugin `rollup-plugin-json`.

### Describe alternatives you've considered

* Users could write their own `rollup_bundle` rules with the help of the exported macros.
* `rollup_bundle` could specify a way to pass a custom `rollup.config.js`
* `rollup_bundle` could specify a way to pass custom plugins.

I still think that adding `rollup-plugin-json` by default should be considered as the primary solution,
as I don't see how it breaks anything and thus adds more towards a "just-works" experience.

**Comment by alexeagle** [comment]
Seems reasonable to me

**Comment by ayinlaaji** [comment]
> # 🚀 feature request
> ### Relevant Rules
> `rollup_bundle`
> 
> ### Description
> In `rollup_bundle`, if I import a package which loads a JSON file via `import` rollup complains with
> 
> ```
> rollup Error: Unexpected token (Note that you need rollup-plugin-json to import JSON files)
> ```
> 
> For example, this happens in `@grpc/grpc-js` via
> 
> ```ts
> import grpc from "@grpc/grpc-js";
> ```
> 
> ### Describe the solution you'd like
> I would like for `rollup_bundle` to automatically make available the plugin `rollup-plugin-json`.
> 
> ### Describe alternatives you've considered
> * Users could write their own `rollup_bundle` rules with the help of the exported macros.
> * `rollup_bundle` could specify a way to pass a custom `rollup.config.js`
> * `rollup_bundle` could specify a way to pass custom plugins.
> 
> I still think that adding `rollup-plugin-json` by default should be considered as the primary solution,
> as I don't see how it breaks anything and thus adds more towards a "just-works" experience.

Did you come up with any solution for this yet ?

**Comment by alexeagle** [comment]
I think this was fixed in #735

**Comment by Globegitter** [comment]
Wouldn't the better solution be to use a `json_to_js` rule similar to what we have written here: https://github.com/ecosia/bazel_rules_nodejs_contrib/blob/master/README.md#json_to_js

Though of the top of the head not a 100% sure how this could apply to external deps.
